### PR TITLE
Custom splash screen

### PR DESCRIPTION
**Pull request by prestosole**
.

### CODE REVIEW

File: packages/clima_ui/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,10 +25,17 @@
             android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize">
             <!-- This keeps the window background of the activity showing
-                 until Flutter renders its first frame. It can be removed if
-                 there is no splash screen (such as the default splash screen
-                 defined in @style/LaunchTheme). -->
-
+     until Flutter renders its first frame. It can be removed if
+     there is no splash screen (such as the default splash screen
+     defined in @style/LaunchTheme). -->
```
Comment (triallax) [inline packages/clima_ui/android/app/src/main/AndroidManifest.xml]: Please return these lines to how they were before.

File: packages/clima_ui/ios/Runner/Info.plist [hunk 2]
```diff
@@ -1,45 +1,47 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
```
Comment (triallax) [inline packages/clima_ui/ios/Runner/Info.plist]: Also return these lines to how they were.

File: packages/clima_ui/ios/Runner/Info.plist [hunk 3]
```diff
@@ -1,45 +1,47 @@
...
+		<array>
+			<string>UIInterfaceOrientationPortrait</string>
+			<string>UIInterfaceOrientationPortraitUpsideDown</string>
+			<string>UIInterfaceOrientationLandscapeLeft</string>
+			<string>UIInterfaceOrientationLandscapeRight</string>
+		</array>
+		<key>UIViewControllerBasedStatusBarAppearance</key>
+		<false/>
+		<key>UIStatusBarHidden</key>
+		<true/>
```
Comment (triallax) [inline packages/clima_ui/ios/Runner/Info.plist]: I believe this will hide the status bar even after the app starts. Is that intentional?

**Review by triallax — CHANGES_REQUESTED** [review]

File: packages/clima_ui/ios/Runner/Info.plist (hunk 3 above)
Comment (prestosole) [reply packages/clima_ui/ios/Runner/Info.plist]: That's intentional.

File: packages/clima_ui/ios/Runner/Base.lproj/LaunchScreen.storyboard [hunk 4]
```diff
@@ -32,6 +38,7 @@
         </scene>
     </scenes>
     <resources>
-        <image name="LaunchImage" width="168" height="185"/>
+        <image name="LaunchImage" width="400" height="400"/>
```
Comment (prestosole) [inline packages/clima_ui/ios/Runner/Base.lproj/LaunchScreen.storyboard]: I'm still not sure about this yet!

**Review by triallax — APPROVED** [review]

**Pull request merged**